java/org/apache/catalina/core/LocalStrings.properties

```diff
@@ -74,7 +74,7 @@ aprListener.aprInitDebug=The Apache Tomcat Native library could not be found usi
 aprListener.aprInitError=The Apache Tomcat Native library failed to load. The error reported was [{0}]
 aprListener.currentFIPSMode=Current FIPS mode: [{0}]
 aprListener.enterAlreadyInFIPSMode=AprLifecycleListener is configured to force entering FIPS mode, but library is already in FIPS mode [{0}]
-aprListener.flags=APR capabilities: IPv6 [{0}], sendfile [{1}], accept filters [{2}], random [{3}].
+aprListener.flags=APR capabilities: IPv6 [{0}], sendfile [{1}], accept filters [{2}], random [{3}], uds [{4}].
```

Comment:
Please uppercase uds in all properties files

Reply:
Done.